plugins/woocommerce/includes/abstracts/abstract-wc-settings-api.php

```diff
@@ -868,8 +868,8 @@ public function validate_text_field( $key, $value ) {
 	 *
 	 * @return string
 	 */
-	public function validate_safe_text_field( string $key, string $value ): string {
-		return wc_get_container()->get( HtmlSanitizer::class )->sanitize( $value, HtmlSanitizer::LOW_HTML_BALANCED_TAGS_NO_LINKS );
+	public function validate_safe_text_field( string $key, $value ): string {
```

Comment:
Nice ... *or* I guess we could keep some level of strictness with:

```suggestion
	public function validate_safe_text_field( string $key, ?string $value ): string {
```

(I keep forgetting but now we have 7.2-language features available to us.) 

Totally optional, though, what we have achieves the same effect.

Reply:
This makes perfect sense. I wasn't sure about it, but it's actually a 7.1 feature so +1 from me.